tools/assets.toit

```diff
@@ -95,6 +96,52 @@ add_asset parsed/cli.Parsed -> none:
         unreachable
     entries[name] = asset
 
+get_cmd -> cli.Command:
+  return cli.Command "get"
+      --options=[
+        cli.OptionEnum "format" ["auto", "binary", "ubjson", "tison"]
+            --default="auto"
+            --short_help="Pick the encoding format.",
+        cli.OptionString "output"
+            --short_name="o"
+            --short_help="Set the name of the output file."
+            --type="file"
+            --required,
+      ]
+      --rest=[
+        cli.OptionString "name"
+            --required,
+      ]
+      --short_help="Get the asset with the given name."
+      --run=:: get_asset it
+
+get_asset parsed/cli.Parsed -> none:
+  input_path := parsed[OPTION_ASSETS]
+  output_path := parsed["output"]
+  name := parsed["name"]
+  format := parsed["format"]
+  entries := load_assets input_path
+  entry := entries.get name
+  if not entry:
+    print "No such asset: $name"
+    exit 1
+  content := entry
+  exception := null
+  if format == "auto":
+    decoded := null
+    catch: decoded = decoded or "$(json.stringify (tison.decode content))\n"
```

Comment:
Would be nice to do this without exceptions.
For tison and maybe ubjson there might be a header. That said, the functionality should be next to the en/decoder and not here...

Reply:
Yeah, I thought about that too. It is super fast to fail, but it sucks to have to use exceptions. Maybe we should have a block -- or maybe we should have a "looks like this might be ubjson/base64/tison/json" methods?